Cargo.toml

```diff
@@ -20,12 +20,14 @@ crate-type = ["staticlib", "rlib"]
 [dependencies]
 assert_matches = "1.5.0"
 atomig = { version = "0.4.0", features = ["derive"] }
+av-data = "0.4.2"
 bitflags = "2.4.0"
 cfg-if = "1.0.0"
 libc = "0.2"
 parking_lot = "0.12.2"
 paste = "1.0.14"
 raw-cpuid = "11.0.1"
+static_assertions = "1"
```

Comment:
Can we keep this like the other dependencies with a full `major.minor.patch`?  We just used what `cargo add` gave us.